vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridHeaderFooterRowIT.java

```diff
@@ -233,6 +233,8 @@ private void assertHeaderTextsAreRendered() {
     public void addHeaderRow_setMultiselect_disableSelection() {
         clickButton("prepend-header");
         clickButton("set-multiselect");
+        WebElement gridVaadinCheckbox = findElement(By.id("selectAllCheckbox"));
+        Assert.assertTrue(String.valueOf(gridVaadinCheckbox.isDisplayed()), true);
```

Comment:
`Assert.assertTrue(...,true)` is pointless, it will always pass.

Reply:
done

Reply:
`Assert.assertTrue(selectAllCheckbox_selectionMode.isDisplayed());` is more readable.
Better yet, if you can provide a meaningful error message:
```java
Assert.assertTrue("The select all checkbox should be displayed", selectAllCheckbox_selectionMode.isDisplayed());
```

Reply:
changed